README.md

```diff
@@ -136,6 +136,7 @@ Bun is an incredibly fast JavaScript runtime, bundler, transpiler and package ma
 
 ## Tools
 
+- [Bunup](https://github.com/arshad-yaseen/bunup) - A high-performance build tool for your libraries built with Bun.
```

Comment:
Maybe it's better like this?

```suggestion
- [Bunup](https://github.com/arshad-yaseen/bunup) - A blazing-fast `tsup` compatible build tool for your libraries built with Bun.
```

I think it should be written that it is compatible with `tsup`